protos/protos/public/modeldb/CommonService.proto

```diff
@@ -113,6 +113,7 @@ message GetUrlForArtifact {
     message Response {
         string url = 1;
         map<string, string> fields = 2;
+        bool multipart_upload_ok = 3;
```

Comment:
The protos changes in this diff are actually already in my protos PR. I guess GitHub isn't showing an accurate diff because these backend PRs are chained on top of each other.

@evgeniilezhnin Could you revert the protos changes in this PR?

Reply:
The protos changes in this diff are actually already in my protos PR. I guess GitHub isn't showing an accurate diff because these backend PRs are chained on top of each other.

@evgeniilezhnin Could you revert the protos changes in this PR?

Reply:
merged the base branch, the changes are gone